folium/plugins/fullscreen.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Fullscreen
+--------------
+
+https://github.com/brunob/leaflet.fullscreen
+
+Adds fullscreen button to your maps.
+"""
+from jinja2 import Template
+
+from branca.element import MacroElement, Figure, JavascriptLink, CssLink
+
+
+class Fullscreen(MacroElement):
+    """
+    Adds a fullscreen button to your map.
+
+    Note: does not work from within an iframe, and because of this, does not
+    work in jupyter notebook
```

Comment:
You mentioned you filed a bug about this, right? Can you add a cross-reference link here so people can follow the status of that?